src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -797,7 +786,7 @@
     <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.SetLastError.cs">
       <Link>Interop\Windows\mincore\Interop.SetLastError.cs</Link>
     </Compile>
-    <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.Globalization.cs">
+    <Compile Condition="'$(EnableDummyGlobalizationImplementation)' != 'true'" Include="..\..\Common\src\Interop\Windows\mincore\Interop.Globalization.cs">
```

Comment:
It would be nice to structure it same way on Unix too - all Common\src\Interop\Unix\System.Globalization.Native entries should be in the Unix full globalization block.